biosimulators_utils/sedml/io.py

```diff
@@ -1372,6 +1372,10 @@ def run(self, filename, validate_semantics=True, validate_models_with_languages=
                 change.target = change_sed.getTarget() or None
                 change.target_namespaces = self._get_namespaces_for_sed_object_targets(change_sed)
                 model.changes.append(change)
+                if isinstance(change_sed, libsedml.SedAddXML) or \
```

Comment:
Similar to the above comment, rather than keeping track of this twice, a method could be added to return the subset of "structural changes".